pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/FormInstance.java

```diff
@@ -85,7 +86,8 @@ public FormInstance(
             Long firstCompletedAt,
             Long lastUpdatedTextTemplateId,
             LocalDateTime activityDefinitionLastUpdated,
-            boolean isFollowup
+            boolean isFollowup,
+            int lastVisitedSection
```

Comment:
See my comment above. Maybe you want to allow this to be a NULL, then you might want an Integer for type.